package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
same

Reply:
removed this file, added this file to .gitignore